benchmarks/dynamo/pr_time_benchmarks/benchmarks/update_hint_benchmark.py

```diff
@@ -34,12 +34,15 @@ def f(a, b):
                 torch._check(x <= self.N)
             return a.split(xs)
 
-        f(self.input, self.splits)
+        for i in range(1000):
+            f(self.input, self.splits)
 
 
 def main():
     result_path = sys.argv[1]
-    Benchmark().enable_instruction_count().collect_all().append_results(result_path)
+    Benchmark().enable_instruction_count().enable_compile_time_instruction_count().collect_all().append_results(
```

Comment:
Are you really proposing that when we have dozens of benchmark we have to manually update each of them to record new metrics we think we want to record?

Reply:
i assume those benchmarks are going to be added one by one? and not all metrics need to be enabled on all of them. 
if someone is adding  a dozen of benchmarks and want to enable all metrics on all of them then he can easily create a function (enableAll()).